go.mod

```diff
@@ -15,28 +15,32 @@ require (
 	github.com/pborman/uuid v1.2.1
 	github.com/spf13/cobra v1.7.0
 	github.com/spf13/viper v1.17.0
-	go.opencensus.io v0.24.0
-	go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc v0.45.0
 	go.uber.org/mock v0.3.0
 	golang.org/x/oauth2 v0.13.0
+	golang.org/x/text v0.14.0
 	google.golang.org/genproto v0.0.0-20231016165738-49dd2c1f3d0b
 	google.golang.org/genproto/googleapis/api v0.0.0-20231016165738-49dd2c1f3d0b
 	google.golang.org/genproto/googleapis/rpc v0.0.0-20231016165738-49dd2c1f3d0b
-	google.golang.org/grpc v1.58.3
+	google.golang.org/grpc v1.59.0
 	google.golang.org/protobuf v1.31.0
 )
 
+require (
+	github.com/go-logr/stdr v1.2.2 // indirect
+	go.opentelemetry.io/otel v1.21.0 // indirect
+	go.opentelemetry.io/otel/metric v1.21.0 // indirect
+	go.opentelemetry.io/otel/trace v1.21.0 // indirect
+)
+
 require (
 	cloud.google.com/go/compute v1.23.1 // indirect
```

Comment:
Why there are 2 require blocks with `// indirect` deps? Was that added manually?

Reply:
must have been a glitch in the visual studio go plugin